br/pkg/restore/split.go

```diff
@@ -431,7 +431,7 @@ func CheckRegionConsistency(startKey, endKey []byte, regions []*RegionInfo) erro
 func PaginateScanRegion(
 	ctx context.Context, client SplitClient, startKey, endKey []byte, limit int,
 ) ([]*RegionInfo, error) {
-	if len(endKey) != 0 && bytes.Compare(startKey, endKey) >= 0 {
+	if len(endKey) != 0 && bytes.Compare(startKey, endKey) > 0 {
 		return nil, errors.Annotatef(berrors.ErrRestoreInvalidRange, "startKey >= endKey, startKey: %s, endkey: %s",
```

Comment:
Update log corresponding to code
```suggestion
		return nil, errors.Annotatef(berrors.ErrRestoreInvalidRange, "startKey > endKey, startKey: %s, endkey: %s",
```